src/js/services/bootstrap.js

```diff
@@ -43,6 +43,8 @@ export default App.extend({
   _getWorkspace(slug) {
     const workspaces = this.currentUser.getWorkspaces();
 
+    if (!workspaces.length) throw 'No workspaces found';
```

Comment:
Previously this function would return `undefined` if there were no workspaces and we'd show the hold up message when `workspace.id` failed... this is a more explict error that we handle explicitly now.